app/util/session_id.py

```diff
@@ -0,0 +1,16 @@
+import uuid
+
+
+class SessionId(object):
+    SESSION_HEADER_KEY = 'Session-Id'
```

Comment:
Just curious -- did you make up this header name or did you find other similar uses of it? All I found was http://www.w3.org/TR/WD-session-id-960221.html